src/nonlinear/room-edits/main-deck/quarantine-bay.s

```diff
@@ -0,0 +1,31 @@
+; Changes: 
+; - Always spawn Hornoad.
+; - Change Hornoad Palette to RoS themed
```

Comment:
This would be better to eithet say Return of Samus, or say that its an unused green/yellow palette